Install issue

<!-- 🔥❓If you have a *QUESTION* about IPFS, please ask on our forum at https://discuss.ipfs.io 🔥❓

The go-ipfs issues are *only* for bug reports and directly actionable feature requests. Please direct all other questions, ideas, or suggestions to our discuss forum. Remember to search the issue tracker for an existing report concerning your issue before posting. If you want to discuss a design idea related to ipfs but not directly related to the go implementation, please file an issue at https://github.com/ipfs/ipfs.
-->

#### Version information:
go version - 1.12.1
<!-- Output From `ipfs version --all`

Please check dist.ipfs.io for a newer version of go-ipfs and update if necessary. Report back if the problem persists.

If you can't run `ipfs version --all` or that command fails, include as much information as you can: IPFS version, computer architecture (e.g., Intel x86 64bit), operating system, etc. -->

#### Type:
getting an error when trying to run `make install`

<!--

- "bug": If what you are filing is a bug.
- "panic": If you'd like to report an application panic, segfault, crash, etc.
- "feature": If you'd like to suggest a feature.
- "enhancement ": If you'd like to suggest an improvement on to existing feature.
- "test failure": If the tests are failing
- "doc": If it's related to missing/incorrect documentation.
- "meta": If its a meta issue about this project/this repository/issue tracker.

Feel free to choose your own category if none of these fit your needs. However, be careful, that may indicate that your issue doesn't belong in this repo.

-->

#### Description:
<!-- This is where you get to tell us what went wrong or your specific feature request. When doing so, please make sure to include *all* relevant information.

When reporting a _bug_, please try to include:
  * What you were doing when you experienced the bug.
  * Any error messages you saw, *where* you saw them, and what you believe may have caused them (if you have any ideas).
  * When possible, steps to reliably produce the bug.

When requesting a _feature_, please be sure to include:
  * Your motivation. Why do you need the feature?
  * How the feature should work.
-->
When running `make install` I am getting an error look like this
```
go version go1.12.1 windows/amd64
bin/check_go_version 1.12
bin/check_go_version: line 10: $'\r': command not found
bin/check_go_version: line 12: $'\r': command not found
bin/check_go_version: line 13: syntax error next to unexpected token «$'{\r'»
'in/check_go_version: line 13: `die() {
make: *** [mk/golang.mk:75: check_go_version] Error 2
```

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,2 +1,10 @@
+# Default to text
+* text eol=lf
+
+# True text
+*.md text eol=auto
+LICENSE text eol=auto
+
+# Binary assets
 assets/init-doc/*          binary
 core/coreunix/test_data/** binary

